cinema/serializers.py

```diff
@@ -71,10 +78,46 @@
         )
 
 
+class TicketSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Ticket
+        fields = "__all__"
+
+
+class TicketListSerializer(TicketSerializer):
+    class Meta:
+        model = Ticket
+        fields = ("row", "seat")
+
+
 class MovieSessionDetailSerializer(MovieSessionSerializer):
     movie = MovieListSerializer(many=False, read_only=True)
     cinema_hall = CinemaHallSerializer(many=False, read_only=True)
+    taken_places = TicketListSerializer
 
     class Meta:
         model = MovieSession
-        fields = ("id", "show_time", "movie", "cinema_hall")
+        fields = ("id",
+                  "show_time",
+                  "movie",
+                  "cinema_hall",
+                  "taken_places",
+                  "tickets_available")
+
+
+class OrderSerializer(serializers.ModelSerializer):
+    tickets = TicketSerializer(many=True,
+                               read_only=True,
+                               allow_empty=False)
```

Comment:
The `tickets` field in `OrderSerializer` is marked as `read_only=True`, which means it cannot be used to create new tickets when creating an order. If the intention is to create tickets as part of the order creation process, `read_only` should be set to `False`.